packages/styles/forms.css

```diff
@@ -14,7 +14,21 @@
   --field-background-color-disabled: #f7f7f7;
   --field-required-text-color: var(--gray-60);
   --field-label-text-color: var(--gray-90);
+  --field-label-error-text-color: var(--error);
   --field-label-description-text-color: var(--gray-60);
+  --icon-inactive-color: var(--gray-90);
+  --icon-active-color: color-mix(
+    in srgb,
+    var(--accent-primary) 20%,
+    transparent
+  );
```

Comment:
GIven that this was only recently introduced we should define a static value so we don't accidentally introduce color contrast issues to users who do not yet support this feature.

Reply:
Just went with the rgba values for now as per [our discussion on slack](https://deque.slack.com/archives/C5MUFQ7UM/p1689889583970209)